### PR TITLE
Add aarch64 support

### LINKED ISSUE #10: Add support for building on Aarch64
With 8u292-b01 the [Aarch64 JIT port](https://bugs.openjdk.java.net/browse/JDK-8257192) has been integrated into JDK 8u mainline. We should add support for building on Aarch64 in our build scripts and publish EA builds for them on Linux.

### PR DESCRIPTION
**Pull request by jhuttana**
Could you please review the changes?
I have verified the build and looks fine.

### CODE REVIEW

File: build-openjdk8.sh [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+    ;;
+  aarch64)
+    echo "/usr/lib/jvm/java-1.7.0-openjdk-1.7.0.221-2.6.18.0.el7_6.aarch64"
+    ;;
+  *)
+    echo "Boot-JDK not fount platform '$arch'" 1>&2
+    exit 1
+    ;;
+  esac
+}
```
Comment (jerboaa) [inline build-openjdk8.sh]: This seems quite ugly and won't work if the version installed on the system isn't exactly the one you are referencing. Instead, lets use a more generic path which will exist irrespective of the underlying architecture. See below.

File: build-openjdk8.sh [hunk 2]
```diff
@@ -43,7 +81,7 @@ build() {
 
   for debug in release slowdebug; do
     bash configure \
-       --with-boot-jdk="/usr/lib/jvm/java-1.7.0-openjdk.x86_64" \
+       --with-boot-jdk="$(boot_jdk)" \
```
Comment (jerboaa) [inline build-openjdk8.sh]: I realize the old boot jdk path wasn't super great to begin with (`/usr/lib/jvm/java-1.7.0-openjdk.x86_64`). Sorry about that. Please use, `--with-boot-jdk="/usr/lib/jvm/java"` which should exist as soon as the `java-1.7.0-openjdk-devel` package is installed. This removes the need to do a switch on the platform architecture in order to figure out the boot JDK path.

On an aarch64 system:
```
# /usr/lib/jvm/java/bin/java -version
java version "1.7.0_221"
OpenJDK Runtime Environment (rhel-2.6.18.0.el7_6-aarch64 u221-b02)
OpenJDK 64-Bit Server VM (build 24.221-b02, mixed mode)
```
On an x86_64 system:
```
# /usr/lib/jvm/java/bin/java -version
java version "1.7.0_261"
OpenJDK Runtime Environment (rhel-2.6.22.1.el6_10-x86_64 u261-b02)
OpenJDK 64-Bit Server VM (build 24.261-b02, mixed mode)
```

**Review by jerboaa — CHANGES_REQUESTED** [review]
@jhuttana Looks mostly good. If we used a more generic boot JDK path, this patch becomes even simpler.

**Comment by jhuttana** [comment]
Addressed review comments. Could you please take a look?
And the build with this change is progressing fine.

**Review by jerboaa — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**